app/helpers/card_content_helper.rb

```diff
@@ -17,6 +17,8 @@ def rating(element)
   end
 
   def rating_color(element)
+    return "rating-green" if rating(element).nil?
```

Comment:
Added guard clause to prevent error if there is no rating assigned to a new cause or effect